tools/reactive-controllers/test/pending-state.test.ts

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+/*
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { expect, fixture, html } from '@open-wc/testing';
+import {
+    HostWithPendingState,
+    PendingStateController,
+} from '@spectrum-web-components/reactive-controllers/src/PendingState.js';
+import sinon from 'sinon';
+import '@spectrum-web-components/progress-circle/sp-progress-circle.js';
+import '@spectrum-web-components/picker/sp-picker.js';
+
+describe('PendingStateController', () => {
+    let host: HostWithPendingState;
+    let controller: PendingStateController<HostWithPendingState>;
+
+    beforeEach(async () => {
+        host = await fixture<HostWithPendingState>(html`
+            <sp-picker aria-label="clickable" pending></sp-picker>
+        `);
+        controller = host.pendingStateController;
+    });
+
+    afterEach(() => {
+        sinon.restore();
+    });
+
+    describe('renderPendingState', () => {
+        it('should change aria-label of host when pending and when not pending', async () => {
+            await host.updateComplete;
```

Comment:
We are not covering if the original `aria-label` is rendered before the component is set to pending.
We might need to render a different fixture for this.

Reply:
Also, could we check what happens if no `aria-label` was provided initially, to check if toggling the `pending` state results in the removal of the `aria-label` as expected?

Reply:
Done!